.github/workflows/CI.yml

```diff
@@ -143,6 +143,14 @@ jobs:
       run: ./run_examples --features "$MOST_FEATURES libc allow-unsafe-code"
     - name: run examples with XCBConnection and dl-libxcb
       run: ./run_examples --features "$MOST_FEATURES libc allow-unsafe-code dl-libxcb"
+
+    # run code generator in nightly builds for coverage
+    - name: Run code generator
+      if: matrix.rust == 'nightly'
+      run: rustup component add llvm-tools-preview
```

Comment:
```suggestion
      run: make
```

Reply:
Thanks!

So much for my copy&paste skills. -.-